src/tags/object/Image/Image.js

```diff
@@ -965,6 +974,11 @@ const Model = types.model({
       self.naturalWidth = naturalWidth;
       self.naturalHeight = naturalHeight;
 
+      self.zoomedPixelSize = {
```

Comment:
better to make it a getter. and anyway it's not good just to assign value to object without defining it at least as `volatile`